package.json

```diff
@@ -30,14 +30,16 @@
   "browser": "dist/geofire.min.js",
   "module": "dist/index.esm.js",
   "typings": "dist/index.d.ts",
+  "utils": "dist/geofire-utils.js",
```

Comment:
I was thinking this would be a separate package.  Is this a normal use for a "distribution"?

Reply:
I was only thinking of pulling the file of a CDN. Do you think we should also allow it to be installed from NPM separately? In that case it'd have to be a new package indeed.

Reply:
If we can figure out how to do another package that would be ideal.  That's how I have it set up for Android.  Then it works better for people installing with `npm` and using bundlers, etc.